packages/block-library/src/navigation/index.php

```diff
@@ -121,6 +121,18 @@ function render_block_core_navigation( $attributes, $content, $block ) {
 
 	unset( $attributes['rgbTextColor'], $attributes['rgbBackgroundColor'] );
 
+	$script_path = __DIR__ . '/navigation/frontend.js';
+
+	if ( file_exists( $script_path ) ) {
```

Comment:
Is this script always needed or only when `isResponsive` is `true`? @pento added a nice guard for the PDF file preview in PHP to load the script on the frontend conditionally, see https://github.com/WordPress/gutenberg/pull/30857/files#r614637000.